packages/core/src/redocly/index.ts

```diff
@@ -9,8 +9,11 @@ const TOKEN_FILENAME = '.redocly-config.json';
 export class RedoclyClient {
   private accessToken: string | undefined;
 
-  constructor() {
+  private readonly redoclyDomain: string;
+
+  constructor(redoclyDomain: string) {
```

Comment:
Breaking change:

Using there: https://github.com/Redocly/redocly-portal/blob/main/engine/plugins/gatsby-source-redocly-filesystem/oas/openapi-cli-bundle.js#L30

Reply:
What do you think about providing a default value for `redoclyDomain` here?

Reply:
I reviewed the remaining PR and it doesn't make sense to provide a default value here. Sorry for bothering